src/Serialization/containers.jl

```diff
@@ -145,6 +187,52 @@ function load_object(s::DeserializerState, T::Type{<:Matrix{S}}, params::NCRing)
   end
 end
 
+function load_object(s::DeserializerState, T::Type{<:Matrix{S}}, key::Int) where S
+  return load_node(s, key) do _
+    load_object(s, T)
+  end
+end
+
+
+################################################################################
+# Multidimensional Arrays
+@register_serialization_type Array "MultiDimArray"
```

Comment:
No objection to the name "MultiDimArray", but is there a reason we don't just call it "Array" (so the same as the underlying type like we do for basically everythin else)?

Reply:
It's to make the a clear distinction for anyone wanting to read this file externally. 
I guess the dims key word does appear but I thought it might help any way to describe it in the name too.
Also to highlight that this is not a Vector (or a matrix for that matter)